translations/ssofact.de.po

```diff
@@ -0,0 +1,177 @@
+msgid "About"
+msgstr "Ãœber das Modul"
+
+#: modules/contrib/ssofact/ssofact.module:24 modules/contrib/ssofact/ssofact.info.yml:0
+msgid "OpenID Connect provider for Newsfactory ssoFACT."
+msgstr "OpenID Connect Anbieter fÃ¼r Newsfactory ssoFACT"
+
+#: modules/contrib/ssofact/ssofact.info.yml:0
+msgid "OpenID Connect Newsfactory ssoFACT Client"
+msgstr "OpenID Connect Newsfactory ssoFACT Client"
+
+#: modules/contrib/ssofact/ssofact.info.yml:0
+msgid "Other"
+msgstr "Andere"
+
+#: modules/contrib/ssofact/ssofact.routing.yml:0
+msgid "OpenID Connect ssoFACT login form redirect page"
+msgstr "OpenID Connect ssoFACT Redirect-Seite fÃ¼r Login-Formular"
+
+#: modules/contrib/ssofact/ssofact.routing.yml:0
+msgid "ssoFACT"
+msgstr "ssoFACT"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "OpenID Connect Newsfactory ssoFACT settings"
+msgstr "OpenID Connect Newsfactory ssoFACT Einstellungen"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Enable client"
+msgstr "Client aktivieren"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Client ID"
+msgstr Client-ID"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Client secret"
+msgstr "Client Secret"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:50
+msgid "Server domain"
+msgstr "Server-Domain"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:58
+msgid "Scope"
+msgstr "Umfang"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:114
+msgid "Email address"
+msgstr "E-Mailadresse"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:129
+msgid "Password"
+msgstr "Passwort"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:137
+msgid "Stay logged-in"
+msgstr "Eingeloggt bleiben"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:145
+msgid "Forgot password?"
+msgstr "Passwort vergessen?"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:151 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:122 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:122
+msgid "Send password reset instructions via email."
+msgstr "Anleitung zur RÃ¼cksetzung des Passworts per Mail senden."
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:165 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:108 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:108
+msgid "Log in"
+msgstr "Einloggen"
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:123 modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:101
+msgid "Your email address"
+msgstr "Ihre E-Mailadresse."
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:144
+msgid "I accept terms and conditions"
+msgstr "Ich akzeptiere die GeschÃ¤ftsbedingungen"
```

Comment:
This is a longer text (see https://app.asana.com/0/633061025239778/732469091269190) and I doubt it makes sense to reverse-translate it into English